cfme/tests/configure/test_access_control.py

```diff
@@ -1618,9 +1630,13 @@ def test_tenant_unique_catalog(appliance, request, catalog_obj):
         'name': catalog_obj.name,
         'description': catalog_obj.description
     })
-    view.add_button.click()
-    view.flash.wait_displayed(timeout=20)
-    assert view.flash.read() == [msg]
+
+    if appliance.version > "5.11":
+        assert view.name.help_block == msg
+    else:
+        view.add_button.click()
+        view.flash.wait_displayed(timeout=20)
+        assert view.flash.read() == [msg]
```

Comment:
**Required** Can you use `view.flash.assert_success_message`?

Reply:
Hey @john-dupuy 
We can not use ```assert_success_message``` as it is error message. Hence used ```assert_message```.

Reply:
Ah, got it, okay that sounds good